### DISCUSSION TITLE
Change Input background color?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#2848] [Q&A] [answered]
Is there a way to set the Input component's background color?

Sometimes a transparent background just doesn't look right, as in this example:

![Chakra Input](https://user-images.githubusercontent.com/55907892/102741848-ac0cf480-4310-11eb-93b2-ac6e1cebb17f.png)

A white background would probably work, here. Maybe. But, I'd like to be able to experiment.

**Comment by chr-ge** [comment, marked as answer]
Just add a `bg` or `bgColor` prop to the input.
```jsx
<Input bg='white' />
```

**Reply by ghost** [reply]
Well, THAT's embarrassing. I really thought I'd tried that before asking the question. But, I must not have, because it works great!

Thanks!
